src/components/PostCard/PostCard.js

```diff
@@ -17,20 +22,81 @@ import {
   Avatar,
   IconButton,
   useDisclosure,
+  Input,
+  InputRightElement,
+  Button,
+  InputGroup,
 } from "@chakra-ui/react";
 import { EditPostModal } from "../EditPostModal/EditPostModal";
-import { deleteUserPost, setPostToBeEdited } from "../../features/postSlice";
+import {
+  addComment,
+  deleteComment,
+  deleteUserPost,
+  editComment,
+  likeAndDislikePost,
+  setPostToBeEdited,
+} from "../../features/postSlice";
```

Comment:
You can explore absolute imports, it makes the code look cleaner
https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8